File: src/main/java/com/meetify/server/Application.java
package com.meetify.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * com.meetify.server
 * Created by kr3v on 02.10.2016.
 */

@SpringBootApplication
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
//        getNearby();
    }
//
//    private static void getNearby() {//throws IOException {
//        String lat = "48.468488";
//        String lon = "35.049684";
//        String radius = "20";
//        try {
//            Optional.ofNullable(new ObjectMapper().readValue(
//                    StringUtils.makeString(HttpRequest.request(lat, lon, radius)),
//                    GooglePlace.class).getResults()).ifPresent(results ->
//                    results.forEach(result -> Optional.ofNullable(result.getPhotos())
//                            .ifPresent(photos -> photos.forEach(photo ->
//                                    photo.setPhotoReference(HttpRequest.photoRefToUrl(photo.getPhotoReference()))))));
//        } catch (IOException e) {
//            e.printStackTrace();
//        }
//    }

}

File: src/main/java/com/meetify/server/Application.kt
package com.meetify.server

import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
import com.meetify.server.model.Id
import com.meetify.server.model.entity.*
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication


/**
 * Основний класс, який має аннотацію SpringBootApplication і відповідний статичний метод.
 * Використовується для запуску сервера.
 *
 * @version 0.0.1
 * @since   0.0.1
 */
@SpringBootApplication
open class Application {
    companion object {
        /**
         * Main-метод, у якому відбувається запуск сервера.
         * @param   args    command-line options
         */
        @JvmStatic fun main(args: Array<String>) {
            SpringApplication.run(Application::class.java, *args)
            val o = jacksonObjectMapper()
            val a = "${o.writeValueAsString(User())}\n${o.writeValueAsString(Place())}" +
                    "\n${o.writeValueAsString(Id())}\n${o.writeValueAsString(Location())}" +
                    "\n${o.writeValueAsString(Photo())}\n${o.writeValueAsString(Login())}"
            println(a)
        }
    }
}